### DISCUSSION TITLE
[Idea] Retrieve type of storage

### DISCUSSION THREAD

**Discussion by AntoninGP** [glpi-project/glpi-agent#427] [Ideas]
### Your idea

Actually, we can't see if a storage is a SSD or a HDD.

**Comment by C-Duv** [comment]
I think the *rotation speed* ("RPM") is `0` for SSD/flash disks.

**Reply by AntoninGP** [reply]
I just had checked, lot of Hard Drives have a "null" rotation speed :/, so isn't working. But the idea was perfect (in theory)

**Comment by g-bougard** [comment]
Hello @AntoninGP 

thank you for your suggestion.

I think this information could be retrieved by consolidating the storage model against the storage type. This is not exactly what you expect as there would have some work for you: make a relation table between your storage model and the storage type to know if it's a HDD or a SSD.

I'm definitively not sure this could be valuable to add this information to current [storages inventory format](https://github.com/glpi-project/inventory_format/blob/b20d75465561a80480da0bbb9acf231c6d9949ee/inventory.schema.json#L1826).

**Reply by AntoninGP** [reply]
I opened the idea, cause i don't know if i'm the only one who need thins kind of info or not.

The table format isn't sustainable for us, cause we use GLPI in order to inventory and follow lifecycle for refurbished products we're selling.
So, new models every week at least.

I know it's not the way GLPI were designed.

I think i'll try with the rotation speed = 0 as @C-Duv advise.
